File: Data Structures & Algorithms/duplicate-integer/submission-1.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> map = new HashSet<int>();
        foreach (var num in nums){
            if (map.Contains(num)){
                return true;
            }
            map.Add(num);
        }
        return false;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        L = 0
        R = 0

        n = len(nums)

        print(n)

        while R < n:
            nums[L] = nums[R]
            while R < n and nums[L] == nums[R]:
                R+=1
            L+=1

        return L
        